CONTRIBUTING.md

```diff
@@ -76,7 +76,8 @@ For your presence to reach the stores, it MUST go through a process on GitHub to
 10. Presences should at least have 1 tag, this is a requirement by design and may be optional in the future.
 11. The `url` field **MUST NOT** include `http://` or `https://`, neither the parameters (e.g. a presence for `https://www.google.com/search?gws_rd=ssl` will only have `www.google.com` in the `url` field).
 12. Descriptions and tags should always be in an array, even when it's only one element. The `url` field, however, should only be a string if it's one domain.
-13. Unstable sites that constantly change APIs/domains, randomize HTML elements or just still being in heavy development are not allowed and will be removed from the store.
-14. The logo and thumbnail on your presence MUST either be `i.imgur.com`, from a CDN that **YOU** own, or from a CDN that gives you permission to **CHANGE** image files.
+13. Tags must not include any spaces, slashes, single/double quotation marks, or any other special characters. Uppercase and lowercase characters as well as numbers are permitted.
```

Comment:
"or any special characters"
which kinda, do you mean these aswell? é ó ë cause then I dont agree.